site/slack.go

```diff
@@ -9,6 +9,7 @@ import (
 
 var bot SlackBot
 var httpRegistered bool = false
+var isValidBot bool = true
```

Comment:
Since is valid bot is a per bot thing, why not set it on the bot struct, rather than globally?